activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+*   Use `Asia/Dubai` for "Abu Dhabi" timezone instead of `Asia/Muscat`
+
+    The "Abu Dhabi" timezone now maps to the canonical `Asia/Dubai` IANA
+    identifier instead of `Asia/Muscat`. This improves consistency with
+    browser timezone detection and correctly represents Abu Dhabi's
+    geographic location in the UAE. Both timezones are functionally
+    identical (UTC+04:00, no DST), so this is not a breaking change.
```

Comment:
Are there other mappings that are out of step with IANA defaults?

The zone may behave the same, but does the mapping change break apps in other ways, e.g. timezone selects, etc?